test/dbus/lvmdbustest.py

```diff
@@ -68,6 +68,42 @@ def lv_n(suffix=None):
 	return g_prefix + rs(8, s)
 
 
+def is_nested_pv(pv_name):
+	return pv_name.count('/') == 3
```

Comment:
This doesn't seem to be very robust. Maybe at least ``> 2`` to support even deeper nesting?